CMakeLists.txt

```
@@ -39,24 +39,24 @@ else()
     project(libsonatareport)
 endif()
 
-option(EXTLIB_FROM_SUBMODULES "Use Git submodules for header-only dependencies" OFF)
-option(${PROJECT_NAME}_CXX_WARNINGS "Compile C++ with warnings as errors" ON)
-option(REPORTS_ENABLE_MPI "Enable MPI-based execution" ON)
-option(SONATA_REPORT_TEST "Enable tests" ON)
+option(SONATA_REPORT_ENABLE_SUBMODULES "Use Git submodules for header-only dependencies" OFF)
+option(SONATA_REPORT_CXX_WARNINGS "Compile C++ with warnings as errors" ON)
```

nitpick : 
```suggestion
option(SONATA_REPORT_ENABLE_WARNING "Compile C++ with warnings as errors" ON)
```
might be better?